adminSiteClient/GdocsIndexPage.tsx

```diff
@@ -1,155 +1,281 @@
+            return false
+        })
+        if (!search.value) return filteredByType
+
+        const fuzzysorted = fuzzysort.go(search.value, searchIndex, {
+            limit: 50,
+            key: "term",
+        })
+
+        const uniqueFiltered = fuzzysorted.reduce(
+            (acc: Set<OwidGdocInterface>, { obj: { gdoc } }) => {
+                if (!gdoc.content.type) return acc.add(gdoc)
+                if (this.filters[gdoc.content.type]) return acc.add(gdoc)
+                return acc
+            },
+            new Set<OwidGdocInterface>()
+        )
+
+        return [...uniqueFiltered]
+    }
+
+    render() {
+        return (
+            <AdminLayout title="Google Docs">
+                <main>
+                    <div className="d-flex justify-content-between mb-3">
+                        <GdocsIndexPageSearch
+                            filters={this.filters}
+                            search={this.search}
+                        />
+                        <button
+                            className="btn btn-primary"
+                            onClick={() =>
+                                this.props.history.push(
+                                    `${this.props.match.path}/add`
+                                )
+                            }
+                        >
+                            <FontAwesomeIcon icon={faCirclePlus} /> Add document
+                        </button>
+                    </div>
+
+                    {this.visibleGdocs.map((gdoc) => (
+                        <div
+                            key={gdoc.id}
+                            className={cx(`gdoc-index-item`, {
+                                [`gdoc-index-item__${gdoc.content.type}`]:
+                                    gdoc.content.type,
+                            })}
+                        >
+                            <div className="gdoc-index-item__content">
+                                {gdoc.content.type ? (
+                                    <span className="gdoc-index-item__type-icon">
+                                        {iconGdocTypeMap[gdoc.content.type]}
+                                    </span>
+                                ) : null}
+                                <Link
+                                    to={`${this.props.match.path}/${gdoc.id}/preview`}
+                                >
+                                    <h5 className="gdoc-index-item__title">
+                                        {gdoc.content.title}
+                                    </h5>
+                                </Link>
+                                <a
+                                    href={`https://docs.google.com/document/d/${gdoc.id}/edit`}
+                                    className="gdoc-index-item__edit-link"
+                                >
+                                    Edit
+                                </a>
```

Comment:
use `GdocEditLink`? 

I would also favor having the edit link in the same place all the time to make it more predictable, especially given that it is one of the main action of this list. E.g. see previous sketches https://github.com/owid/owid-grapher/issues/1613#issuecomment-1241875170